source/blog/2021-12-07-announcing-hanami-200alpha4.html.markdown

```diff
@@ -0,0 +1,110 @@
+---
+title: Announcing Hanami v2.0.0.alpha4
+date: 2021-12-07 13:15:19 UTC
+tags: announcements
+author: Luca Guidi
+image: true
+excerpt: >
+  Streamlined source directories, more flexible settings, better integrated actions, and the beginning of a monthly release cadence.
```

Comment:
n.b. I think this is leftover from the last announcement

Reply:
Fixed it, thanks 😃